### PR TITLE
 Infers common credential helper names from target registry.

### LINKED ISSUE #20: Infer common credential helper names from target registry.
From https://github.com/GoogleCloudPlatform/minikube-build-tools-for-java/issues/115

For example, `gcr.io` should use `docker-credential-gcr`.

### PR DESCRIPTION
**Pull request by coollog**
Fixes #20

### CODE REVIEW

**Comment by coollog** [comment]
Should release v0.1.1 after this is merged.

File: jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/BuildImageMojo.java [hunk 1]
```diff
@@ -55,6 +55,20 @@
...
+  @VisibleForTesting
+  @Nullable
+  static String inferCredentialHelperName(String registry) {
+    if (registry.endsWith("gcr.io")) {
+      return "gcr";
+
+    } else if (registry.endsWith("amazonaws.com")) {
+      return "ecr-login";
+    }
+    // TODO: Add more common credential helpers.
```
Comment (loosebazooka) [inline jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/BuildImageMojo.java]: nit: this TODO seems unnecessary.

Comment (coollog) [reply jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/BuildImageMojo.java]: This is mostly as a note to add more inferences such as Azure's -acr-* credential helpers. Is it standard practice to not have a note like this?

Comment (loosebazooka) [reply jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/BuildImageMojo.java]: Yeah, everyone uses them differently. I would think it's better to file a bug than leave this here, I tend to lose track of TODOs.

Comment (coollog) [reply jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/BuildImageMojo.java]: Filed https://github.com/google/jib/issues/79

**Pull request merged**